packages/pentaho/src/Canvas/ToolbarTabs/ToolbarTabs.styles.tsx

```diff
@@ -1,15 +1,16 @@
 import { tabClasses } from "@mui/base/Tab";
-import {
-  baseInputClasses,
-  createClasses,
-  theme,
-} from "@hitachivantara/uikit-react-core";
+import { createClasses, theme } from "@hitachivantara/uikit-react-core";
+
+import { toolbarTabEditorClasses } from "./ToolbarTabEditor";
 
-export const ICON_WIDTH = 32;
 export const MIN_TAB_WIDTH = 120;
-const MAX_TAB_WIDTH = 220;
+export const MAX_TAB_WIDTH = 220;
+export const DROPDOWN_MENU_WIDTH = 64;
 const TAB_HEIGHT = 32;
-const TAB_LABEL_HEIGHT = 24;
+const CLOSE_ICON_SIZE = 32;
+const TAB_ICON_SIZE = 16;
+const TAB_PADDING = theme.space.xs;
+const TAB_COLOR = "#EFF6FF"; // TODO - change HEX for token when available (dark/light)
```

Comment:
This doesn't look good in dark mode:

![image](https://github.com/user-attachments/assets/a1ea85e7-3221-4c5b-8f99-89a8a234036f)

Don't we have a dark color that we can use?
If we do, we can define a CSS var (eg. `--tabColor`) with a fallback value, and override it in the `pentahoPlus` theme under `components.HvCanvasToolbarTabs.classes.root`
If we don't... I think we should decide a color from the palette ourselves, or keep what we previously had

What do you think?

Reply:
I'll address this in another PR